docs/config-file/v2.rst

```diff
@@ -8,15 +8,26 @@ Here is an example of how this file looks like:
 
 .. code:: yaml
```

Comment:
Instead of the above line saying,

> Here is an example of how this file looks like:

what about saying something like "Here is a minimum example configuration file:" or some better phrase than that one but making it clear that it's the _minimum_ inviting the user to copy and paste as is other than keep reading the whole page.

Reply:
I think there's a few changes so that the YAML docs are better "at a glance". I think that should be a separate PR but I can take care of this small change here.